src/cockpit/389-console/src/lib/replication/replTasks.jsx

```diff
@@ -429,30 +420,30 @@ export class ReplRUV extends React.Component {
                 </div>
                 <hr />
                 <h4 className="ds-margin-top-xlg">Create Replica Initialization LDIF File</h4>
```

Comment:
I wonder if should at least display the server LDIF directory path here...
Ideally, it will be list the LDIFs from the directory, IMO.
 What do you think?

Reply:
I agree it can be improved.  I like the table idea.  Ok let me look into it...